packages/httpServer/dependencies/swagger/createSwaggerDefinition/index.js

```diff
@@ -4,12 +4,14 @@ import httpStatus from 'http-status-codes'
 
 const PATH_DELIMITER = '/'
 const PARAM_DELIMITER = ':'
+const REQUIRED = 'required'
 const SCHEMA_TYPES = {
   QUERY_PARAMS: 'queryParams',
   RESPONSE_BODY: 'responseBody',
-  REQUEST_BODY: 'requestBody'
+  REQUEST_BODY: 'requestBody',
+  PATH_PARAMS: 'pathParams'
 }
-const { REQUEST_BODY, RESPONSE_BODY, QUERY_PARAMS } = SCHEMA_TYPES
+const { REQUEST_BODY, RESPONSE_BODY, QUERY_PARAMS, PATH_PARAMS } = SCHEMA_TYPES
```

Comment:
We could just make these individual variables, no need to do this weird thing with SCHEMA_TYPES since we're not exporting it anywhere.